mapboxgl/viz.py

```diff
@@ -276,3 +276,33 @@ def add_unique_template_variables(self, options):
             clusterRadius=self.clusterRadius,
             clusterMaxZoom=self.clusterMaxZoom
         ))
+
+
+class ImageViz(MapViz):
```

Comment:
@vincentsarago could we call this class an `ImageLayer`? it looks layer-like to me and I think `Layer` could tell a user a lot about how to use the class.

Reply:
Right now it's a proper MapViz, but I think at the end we could make it a layer meaning that instead of having a ImageViz, we could add a `addImage` function to the `MapViz` class.

Reply:
Agree, a change to use `layer` nomenclature makes sense here.  I'd propose a renaming of the API from `*Viz` to `*Layer` in a separate PR from this one, as a part of https://github.com/mapbox/mapboxgl-jupyter/issues/34.